Modix/ClientApp/src/components/UserLookup/UserSearch.vue

```diff
@@ -2,10 +2,16 @@
     <section class="section">
         <div class="message-narrow">
             <div class="field">
-                <label class="label">User ID</label>
+                <label class="label is-large">Tell us their username or ID</label>
                 <div class="control">
-                    <input :class="inputClass" type="text" placeholder="Enter a User ID..." v-model="userId" />
+                    <Autocomplete @select="selectedUser = $event"
+                                  :serviceCall="userServiceCall" placeholder="Enter a username or an ID">
+                        <template slot-scope="{entry}">
+                            <TinyUserView :user="entry" />
+                        </template>
+                    </Autocomplete>
                 </div>
+                <p class="help is-danger"></p>
```

Comment:
Does this actually show up on the page?

Reply:
no clue dude, that bit was basically just copy paste